contributing.md

```diff
@@ -13,18 +13,33 @@ machine to develop Bazel and, when you've made a patch, how to submit it.</p>
 In general, we prefer contributions that fix bugs or add features (as opposed to
 stylistic, refactoring, or "cleanup" changes).
 
-Please check with us on the [dev
-list](https://groups.google.com/forum/#!forum/bazel-dev) before investing a lot
-of time in a patch. Meet Bazel maintainers and other contributors on
-[Slack](https://slack.bazel.build).
+### What issues can I work on?
+
+In general, if there is a Bazel team member assigned to an issue, someone is
```

Comment:
I would say "if there is someone" instead of "if there is a Bazel team member" - generally speaking anyone can work on any issue, not just Bazel team members.